docs/user-guide/docs-source/setup.md

```diff
@@ -0,0 +1,123 @@
+# Setup
+
+
+
+## Initial Setup (One-Time)
+
+Complete the following setup before using Ambit for the first time.
+
+>  ⏱ Initial setup takes about 20 minutes.
+
+> 🛑 These instructions assume you already have Unreal Engine 4.27 installed. If you don't, you can [download Unreal Engine for free](https://www.unrealengine.com/en-US/download). Choose the Unreal Engine license that is appropriate for your team. Ambit will work with any Unreal Engine license.
+
+### 1. Download the Ambit plugin for UE4
```

Comment:
This section is more or less repeated in `Per Project Setup` and makes for an awkward read, especially since it states that we'll return to the Ambit plugin in later steps and then gives the same instructions we have here. We should re-organize these two or re-word them.

Reply:
I've made some changes in response to your feedback that you'll see in the next update. This is a bit of a tricky one to handle gracefully, but I _think_ I've made it a little smoother using your feedback.